notification/tests.py

```diff
@@ -15,7 +15,29 @@ def test_delete_notification_from_database(self, notification):
         with pytest.raises(Notification.DoesNotExist):
             Notification.objects.get(pk=notification.pk)
 
-    def test_mark_as_read_notification(self, notification):
+    def test_update_notification_message(self, notification):
+        new_notification = notification
```

Comment:
Are you asking about the test itself, or about the `new_notification = notification`?

If you refer to the function itself, I thought that I should test any change in the data, for any future features that would be.

If you refer to the assignment itself:
The `new_notification` holds a copy of `notification`. 
I want to edit the values of the original variable, and to make sure that I have a copy of the old one aswell.
If I'll change the original data, and for some reason it would get a different ID, the test would fail at `assert new_notification.id == notification.id == updated_notification.id`.
Yes, I could save the ID itself instead of the `new_notification` variable, but I think that both serve the same goal and thus I've saved the older copy.

Reply:
I was discussing the `new_notification `variable. I believe it functions as a pointer to the exact same object rather than being a copy of `notification`. Therefore, I think it doesn't serve your logic

Reply:
Yup. I guess I was mistaken at that part. I thought that with this assignment I will have a copy of notification, and not a pointer. Will be fixed asap.